backend/app/main.py

```diff
@@ -146,3 +149,23 @@ async def get_total_population(
     stmt = text("SELECT * from get_total_population(:document_id)")
     result = session.execute(stmt, {"document_id": document_id})
     return [ZonePopulation(zone=zone, total_pop=pop) for zone, pop in result.fetchall()]
+
+
+@app.get("/api/gerrydb/views", response_model=list[GerryDBViewPublic])
+async def get_projects(
+    *,
+    session: Session = Depends(get_session),
+    offset: int = 0,
+    limit: int = Query(default=100, le=100),
+):
+    gerrydb_views = session.exec(
+        select(
+            GerryDBTable.name.label("table_name"),
+            GerryDBTiles.s3_path.label("tiles_s3_path"),
+        )
+        .join(GerryDBTiles, GerryDBTable.uuid == GerryDBTiles.table_uuid, isouter=True)
```

Comment:
(o) I think we had talked about upstream validation and inner joins? or is there a different case where we might have unmatched rows?

Reply:
I was wondering about this and left it open for now. I was thinking maybe there's a case where we want to generate some metrics on a GerryDB view that isn't itself associated with a tileset?

Reply:
got rid of join because I merged the tables